data/settings_mapping.json

```diff
@@ -348,7 +348,8 @@
             "big_poe_count",
             "easier_fire_arrow_entry",
             "fae_torch_count",
-            "ruto_already_f1_jabu"
+            "ruto_already_f1_jabu",
+            "tcg_requires_lens"
```

Comment:
Timesavers is probably the wrong section for this. I'd suggest moving it to Misc.

Reply:
done